### PR TITLE
[deps] update http-client to 4.5.13

### PR DESCRIPTION
**Pull request by kares**
Mostly wanted to see how this is doing against the CI - seems to be all :green_circle: 

Let me know what else is needed.

### CODE REVIEW

**Comment by davishmcclurg** [comment]
hey @kares! if this gets merged, I think we can use `TrustAllStrategy` directly here: https://github.com/cheald/manticore/pull/100

File: lib/manticore_jars.rb [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+  require 'org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar'
+  require 'org/apache/httpcomponents/httpmime/4.5.13/httpmime-4.5.13.jar'
+end
 
-require_jar("commons-logging", "commons-logging", "1.2")
-require_jar("org.apache.httpcomponents", "httpmime", "4.5.2")
-require_jar("commons-codec", "commons-codec", "1.10")
-require_jar("org.apache.httpcomponents", "httpclient", "4.5.2")
-require_jar("org.apache.httpcomponents", "httpcore", "4.4.4")
+if defined? Jars
```
Comment (davishmcclurg) [inline lib/manticore_jars.rb:12]: Why the two different ways of loading jars now?

Comment (kares) [reply lib/manticore_jars.rb:12]: this file is autogenerated, the change is likely due newer jar_dependencies

**Comment by kares** [comment]
Hey @olleolleolle do you know who should we ping to get this and the follow-up https://github.com/cheald/manticore/pull/100 reviewed?

**Comment by kares** [comment]
Hey @cheald any chance we could consider reviewing merging and pushing this one?
I'd be happy to take over some (easy) maintenance tasks as/if needed.

**Comment by olleolleolle** [comment]
@kares 👋 This is great - and - the README notes that the preferred & canonical way to get changes into this project is at https://gitlab.com/cheald/manticore/-/merge_requests - is that true, @cheald? There _seems_ to be an issue that matches #100 there, if I squint a little. https://gitlab.com/cheald/manticore/-/issues/53

**Comment by cheald** [comment]
Apologies, folks, I've been lax in keeping up on this one. I'm not using JRuby actively anymore, so I don't have the setup to immediately test it, but I'm happy to merge it if you all are happy with it!

**Comment by cheald** [comment]
> Hey @cheald any chance we could consider reviewing merging and pushing this one? I'd be happy to take over some (easy) maintenance tasks as/if needed.

If you'd like to take over maintenance, I'm more than happy to hand it off! I'm unfortunately not in the JRuby arena much these days, so I'd love for someone with an active interest in this project to be managing it!

**Pull request merged**